src/components/componentStyles.css

```diff
@@ -64,3 +64,59 @@ button {
 .zero {
   grid-column: 1 / span 2;
 }
+
+.navBar {
```

Comment:
- [ ] **OPTIONAL**
The site has a small overflow that makes your great design looks incomplete, would you mind fixing it? 😉